msg_broker/src/index.test.ts

```diff
@@ -45,3 +45,4 @@ test('should read message from queue successful', async () => {
   expect(mock.info).toHaveBeenCalledWith('[PCAP] -', 'Message-01')
   expect(mock.info).toHaveBeenCalledWith('[DHCP] -', 'Message-01')
 })
+/* eslint-enable @typescript-eslint/explicit-function-return-type */
```

Comment:
bof bof (english)

Reply:
It is a typo defined by eslint https://eslint.org/docs/2.13.1/user-guide/configuring